src/dotnet/Chat.UI.Blazor/Components/ChatMessageEditor/AttachDropdown.razor

```diff
@@ -2,14 +2,14 @@
 @using ActualChat.UI.Blazor.Components
 @using ActualChat.Hosting
 
-<Dropdown MenuClass="" ButtonClass="btn-transparent">
+<Dropdown ButtonClass="attach-btn btn-square" Class="flex items-center justify-end">
```

Comment:
Why `attach-btn`, not `attach-dropdown`?

Reply:
There is no agreement to add `-btn` or whatever else suffix to the component's main selector.